thirdparty/github.go

```diff
@@ -331,7 +331,7 @@ func getGithubClient(token, caller string, config retryConfig) *githubapp.GitHub
 	// to the database. Otherwise we're running in the agent and we should use an in-memory cache.
 	// We could stop casing on this if we were to stop calling out to GitHub from the agent.
 	if evergreen.GetEnvironment() != nil {
-		cacheTransport.Cache = &cache.DBCache{}
+		cacheTransport.ContextCache = &cache.DBCache{}
```

Comment:
The dependency update manifests here, in a new field exposed called `ContextCache`. The implementations are a bit different now, in which they accept a context and return an error.